pp/integrations/plugin-packs/procedures/virtualization-vmware2-vcenter-generic.md

```diff
@@ -759,290 +759,290 @@ yum install centreon-plugin-Virtualization-Vmware2-Connector-Plugin
+| CRITICALSNAPSHOT | Critical threshold                                                                                                             |                   |             |
+| WARNINGSTATUS    | Define the conditions to match for the status to be WARNING. You can use the following variables: %{accessible}                |                   |             |
+| CRITICALSTATUS   | Define the conditions to match for the status to be CRITICAL. You can use the following variables: %{accessible}               |                   |             |
+| WARNINGTOTAL     | Warning threshold                                                                                                              |                   |             |
+| CRITICALTOTAL    | Critical threshold                                                                                                             |                   |             |
+| EXTRAOPTIONS     | Any extra option you may want to add to the command (e.g. a --verbose flag). All options are listed [here](#available-options) | --verbose         |             |
 
 </TabItem>
 <TabItem value="Datastore-Usage-Global" label="Datastore-Usage-Global">
 
-| Macro               | Description                                                                                                                                                | Default value     | Mandatory   |
-|:--------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
-| FILTER              | The connector will only take into account alerts coming from the datastores listed here                                                                                                                                     | .*                |             |
-| UNIT                |                                                                                                                                                            | %                 |             |
+| Macro               | Description                                                                                                                    | Default value     | Mandatory   |
+|:--------------------|:-------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTER              | The connector will only take into account alerts coming from the datastores listed here                                        | .*                |             |
+| UNIT                |                                                                                                                                | %                 |             |
```

Comment:
```suggestion
| UNIT                | Set this option to '%' if you want the thresholds to be interpreted as a percentage of the maximum possible value instead of as an absolute threshold.  | %                 |             |
```